app/assets/stylesheets/base.scss

```diff
@@ -10,14 +10,35 @@ a {
 
 h1 {
   color: $sul-h1-font-color;
-  font-size: 2.3em;
+  font-size: 2.1em;
   letter-spacing: -0.05em;
```

Comment:
Kill letter-spacing - it's only relevant to SearchWorks when using deja-vu font.